### PR TITLE
Various config setting description improvements

### PR DESCRIPTION
**Pull request by johnnovak**

### CODE REVIEW

File: src/dosbox.cpp [hunk 1]
```diff
@@ -815,11 +815,12 @@ void DOSBOX_Init()
 	pint = secprop->Add_int("voodoo_perf", only_at_start, 1);
 	pint->SetMinMax(0, 3);
 	pint->Set_help("Performance optimisations to use when emulating the 3dfx Voodoo card:\n"
-	               "Note: emulation is software-based and does not use host-level OpenGL calls.\n"
 	               "   0:  No optimizations.\n"
 	               "   1:  Multi-threading (default).\n"
 	               "   2:  Disable bilinear filtering.\n"
-	               "   3:  All optimizations (both 1 and 2).");
+	               "   3:  All optimizations (both 1 and 2).\n"
+	               "Notes: Voodo emulation is software-based and does not use host-level\n"
```
Comment (shermp) [inline src/dosbox.cpp:822]: It's a single note, so `Note:`?

**Review by shermp — APPROVED** [review]
I agree, moving the note to the end reads better.

File: src/dosbox.cpp (hunk 1 above)
Comment (johnnovak) [reply src/dosbox.cpp:822]: We use `Notes:` everywhere.

Comment (shermp) [reply src/dosbox.cpp:822]: Fair enough. If it's the same everywhere else, who am I to argue?

Comment (johnnovak) [reply src/dosbox.cpp:822]: Ok, on second thought, you're right @shermp . I'll refactor all existing descriptions in an additional commit so it's consistent:
- For singles note, it's `Note`
- For multiple notes in a bulleted list, it's `Notes`

Thanks for pointing this out! 🎉

**Review by kcgen — APPROVED** [review]
👍

**Comment by johnnovak** [comment]
Okay, I've added a few more minor cleanups/improvements to this @shermp @kcgen. I'll stop here 😄

**Comment by shermp** [comment]
I think it looks Ok. Since you've expanded it, perhaps make the PR title less voodoo specific?

**Comment by johnnovak** [comment]
> I think it looks Ok. Since you've expanded it, perhaps make the PR title less voodoo specific?

Done.

**Pull request merged**